### ISSUE TITLE
Cannot "shpc config edit" if there's a bug in settings.yaml

### ISSUE THREAD

**Issue by marcodelapierre** [singularityhub/singularity-hpc#398]
If invalid syntax is entered in `settings.yaml`, attempts to edit it with `shpc config edit` will fail.

It would be good if this specific command could be exempted from checking `settings.yaml` (after all, I don't think it needs it), so that it can be used to fix the file.

E.g. here I used an invalid value for module_sys:
```
$ shpc config edit
Traceback (most recent call last):
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/bin/shpc", line 11, in <module>
    load_entry_point('singularity-hpc', 'console_scripts', 'shpc')()
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/shpc/client/__init__.py", line 349, in run_shpc
    main(args=args, parser=parser, extra=extra, subparser=helper)
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/shpc/client/config.py", line 13, in main
    cli = get_client(quiet=args.quiet, settings_file=args.settings_file)
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/shpc/main/__init__.py", line 26, in get_client
    settings = Settings(kwargs.get("settings_file"))
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/shpc/main/settings.py", line 149, in __init__
    self.validate()
  File "/group/pawsey0001/mdelapierre/PSS/shpc-tests/shpc-pr-vsoch/shpc_fixes_8jun/shpc/main/settings.py", line 40, in validate
    jsonschema.validate(instance=self._settings, schema=shpc.main.schemas.settings)
  File "/group/pawsey0001/mdelapierre/.local/lib/python3.6/site-packages/jsonschema/validators.py", line 934, in validate
    raise error
jsonschema.exceptions.ValidationError: 'lua' is not one of ['lmod', 'tcl', None]

Failed validating 'enum' in schema['properties']['module_sys']:
    {'enum': ['lmod', 'tcl', None], 'type': 'string'}

On instance['module_sys']:
    'lua'
```

**Comment by vsoch** [comment]
Will fix this asap!

**Comment by vsoch** [comment]
All set! https://github.com/singularityhub/singularity-hpc/pull/397/commits/0e7d05e134b4c53de83d3d10d24ce5804586ab52

**Comment by marcodelapierre** [comment]
cool!

**Comment by vsoch** [comment]
Closed with #397
